apps/aecore/src/aec_next_nonce.erl

```diff
@@ -0,0 +1,17 @@
+-module(aec_next_nonce).
+
+-export([for_account/0]).
+
+for_account() ->
```

Comment:
this will need to talk to code in mempool PR

Reply:
Agreed.